neoutils/connection.go

```diff
@@ -50,29 +51,33 @@ func DefaultConnectionConfig() *ConnectionConfig {
 	}
 }
 
-func Connect(neoURL string, conf *ConnectionConfig) (NeoConnection, error) {
+func Connect(neoURL string, conf *ConnectionConfig, log *logger.UPPLogger) (NeoConnection, error) {
+	if log == nil {
```

Comment:
I prefer throwing an error if the logger is not valid.

Reply:
The idea is to make the logger optional; i.e. initialize it if not provided by the client.